keras_retinanet/bin/train.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
```

Comment:
Since this file moved into the `keras-retinanet` package we should use relative imports. However, this file can be used as library and as script. That means special care is required to make relative imports work in both cases.

Reply:
Would you be so kind as to look into this issue?

Reply:
Additionally, what about those using virtualenvs? Would this overwrite the default python binary to execute? If hashbangs are going to be used, should they be consistent with all python files?

Reply:
using `/usr/bin/env` to execute python ensures that it is looked up using $PATH. To my knowledge, virtualenvs use $PATH to change the invoked python interpreter, so this shebang should work fine with virtualenv.

Reply:
I added two commits to use relative imports in the scripts/modules in bin. I pretty much did what [PEP-366](https://www.python.org/dev/peps/pep-0366/) suggests. Sadly this includes fiddling with `sys.path`.

Relevant excerpt from PEP-366:

>When the main module is specified by its filename, then the `__package__` attribute will be set to None. To allow relative imports when the module is executed directly, boilerplate similar to the following would be needed before the first relative import statement:

```
if __name__ == "__main__" and __package__ is None:
    __package__ = "expected.package.name"
```

> Note that this boilerplate is sufficient only if the top level package is already accessible via `sys.path`. Additional code that manipulates `sys.path` would be needed in order for direct execution to work without the top level package already being importable.